content/src/components.ts

```diff
@@ -41,8 +42,12 @@ export async function initComponentsWithEnv(env: Environment): Promise<AppCompon
   const repository = await RepositoryFactory.create({ env, metrics })
   const logs = createLogComponent()
   const fetcher = createFetchComponent()
+  const contentStorageFolder = path.join(env.getConfig(EnvironmentConfig.STORAGE_ROOT_FOLDER), 'contents')
+  const tmpDownloadFolder = path.join(contentStorageFolder, '_tmp')
+  await fs.promises.mkdir(tmpDownloadFolder, { recursive: true })
```

Comment:
Does this fail if the folder already exists?

Reply:
Yes, it fails, @pedrotambo can we fix that and add a try catch
https://stackoverflow.com/questions/21194934/how-to-create-a-directory-if-it-doesnt-exist-using-node-js

Reply:
It fails but only when `recursive` is false.